### ISSUE TITLE
<|fim_pad|> or <|endoftext|> for padding during continued pre-training/fine-tuning with FIM?

### ISSUE THREAD

**Issue by nathan-az** [QwenLM/Qwen2.5-Coder#423]
Both appear in the tokenizer, but it's not clear to me which to use for padding examples in batches (or just when padding to consistent length for specific kernels).

**Comment by cyente** [comment]
you could use any of them
